### PR TITLE
rename readme.htm to readme.html

### PR DESCRIPTION
**Pull request by arihant2math**
Changed the file.d to file.html and renamed readme.htm.

### CODE REVIEW

**Review by mhammond — CHANGES_REQUESTED** [review]
Thanks! However, the `.d` files are processed by autoduck and included in the .chm help file shipped with the distribution, so most of the changes there should be undone. It would be great if we could work out how to get the generated .html and put that up somewhere, but for now just the typos etc in that file should be changed

**Comment by arihant2math** [comment]
@mhammond I fixed that now.

File: win32/help/file.d [hunk 1]
```diff
@@ -14,23 +12,18 @@ the ability to do exclusive or shared locking. However, what it does
...
 <nl>The basic procedure for doing this is to first call Createfile to 
 give you a filehandle. Then call LockfileEx with the filehandle.
 Do whatever to the file. Call UnlockfileEx. Then close the filehandle.
 (Some of you may want to close the filehandle to kill the locks, it
 doesn't work that way with win32, at least according to the msdn)
-
 Below is a class called Flock, which gives you exclusive/shared
 locking with non-blocking/blocking abilities. If you can think of any
 optimizations or changes, be sure to let me know. 
-
```
Comment (mhammond) [inline win32/help/file.d:27]: Have you checked how this now renders? I'm worried that removing many of these blank lines will hurt readability - especially between human text and examples. What is your motivation for touching this file at all?

**Review by mhammond — COMMENTED** [review]
Thanks for the update, but I'd like to understand why you are touching file.d at all?

**Comment by arihant2math** [comment]
@mhammond any update on this?

**Comment by arihant2math** [comment]
I reverted the commits

**Comment by arihant2math** [comment]
@mhammond Any updates on this?

**Pull request closed without merging**